src/main/webapp/assets/modules/salary-report/sr-similar-job.js

```diff
@@ -11,12 +11,17 @@ techlooper.directive("srSimilarJob", function (jsonValue, connectionFactory, $ti
       scope.doJobAlert = function ($event) {
         $event.preventDefault();
 
-        $('.email-similar-jobs-block').slideDown("normal");
+        $('.email-similar-jobs-block').show();
         focus("emailJobAlert");
 
+        if (scope.jobAlert) {
+          emailSuggestion =  scope.jobAlert.email || "";
+        }
         scope.jobAlert = angular.copy(scope.salaryReview);
         scope.jobAlert.frequency = timeToSends[0].id;
-        scope.jobAlert.email = emailSuggestion;
+        if (!scope.jobAlert.email) {
+          emailSuggestion.length > 0 && (scope.jobAlert.email = emailSuggestion);
```

Comment:
Line is too long.<br>Expected an assignment or function call and instead saw an expression.